tests/cases/fourslash/goToImplementationInterface_07.ts

```diff
@@ -19,7 +19,7 @@
 //// let x7: (new() => Foo) = [|class { hello ()    { /**constructorType*/} }|];
 //// let x8: Foo[]          = [|[{ hello ()         { /**arrayType*/} }]|];
 //// let x9: { y: Foo }     = [|{ y: { hello ()     { /**typeLiteral*/} } }|];
-//// let x10 = [|{|"parts": ["(","anonymous local class",")"], "kind": "local class"|}class implements Foo { hello() {} }|]
+//// let x10 = [ [|{|"parts": ["(","local class",")"," ", "(Anonymous class)"], "kind": "local class"|}class implements Foo { hello() {} }|] ]
```

Comment:
Presumably, someone made `anonymous local class` intentionally - maybe because the new behavior is longer. Is there any test that still shows it in use?

Reply:
Nope, this is the one. I assume they made it for the same reason my recent PR special cases class expressions; getSymbolAtLocation didn't support them.

It's not too hard to restore the old text, but I personally think the new text is better. I'll defer to reviewers' intuitions about what's readable, though.

Reply:
My vote is to maintain either `(local anonymous class)` or just switch entirely to `(Anonymous class)`

Reply:
Added in #14001